eclipse-mat.json

```diff
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://www.eclipse.org/mat/",
+    "license": "EPL-1.0",
+    "version": "1.8.0.20180604",
+    "architecture": {
+        "64bit": {
+            "url": "http://download.eclipse.org/mat/1.8/rcp/MemoryAnalyzer-1.8.0.20180604-win32.win32.x86_64.zip",
+            "hash": "d4f1f56c4da5d2b4ed2a56de642aa0be5425b47776d085436f8b3a83b08c60b4"
+        },
+        "32bit": {
+            "url": "http://download.eclipse.org/mat/1.8/rcp/MemoryAnalyzer-1.8.0.20180604-win32.win32.x86.zip",
+            "hash": "1b240d73b7e7becd2f31e7f3e86e932bfc8a65c92fef234106ae815cb64f3bf3"
+        }
+    },
+    "extract_dir": "mat",
+    "persist": "workspace",
+    "shortcuts": [
+        [
+            "MemoryAnalyzer.exe",
+            "Eclipse Memory Analyzer"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.eclipse.org/mat/downloads.php",
+        "re": "MemoryAnalyzer-([\\d.]+)-win32"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "http://download.eclipse.org/mat/1.8/rcp/MemoryAnalyzer-$version-win32.win32.x86_64.zip"
```

Comment:
That 1.8 inside URL is changing with Version? 
I suggest to use `$majorVersion.$minorVersion/rcp`